From b80cd89b145b464813b165d7c4ad90315499c143 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sun, 2 Sep 2012 23:40:08 -0400
Subject: [PATCH] Fix libcanberra.so: cannot open shared object file: No such
 file

For one thing, we need the GTK3 version, not the GTK2 version.

And then let's go with the file that's installed.

I believe this will break if libcanberra ever needs to do a
soname bump though.

https://bugzilla.gnome.org/show_bug.cgi?id=683235
---
 gnomeclocks/pycanberra.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnomeclocks/pycanberra.py b/gnomeclocks/pycanberra.py
index 36a0245..1cb8d60 100644
--- a/gnomeclocks/pycanberra.py
+++ b/gnomeclocks/pycanberra.py
@@ -502,7 +502,7 @@ _libHandle = None
 def GetApi():
    global _libHandle
    if not _libHandle:
-      _libHandle = CDLL("libcanberra.so")
+      _libHandle = CDLL("libcanberra-gtk3.so.0")
    return _libHandle
 
 
